src/drt/src/dr/FlexDR_init.cpp

```diff
@@ -3087,6 +3087,22 @@ void FlexDRWorker::initMazeCost_planarTerm(const frDesign* design)
       // term no bloat
       switch (obj->typeId()) {
         case frcBTerm: {
+          auto bterm = static_cast<frBTerm*>(obj);
```

Comment:
Does this have anything to do with repair_pdn_vias or is it just two changes in one PR?

Reply:
two changes in one PR. This is the part for fixing the blocked edges above bterm